docs/swupd.1.rst

```diff
@@ -189,6 +189,26 @@ SUBCOMMANDS
 
     Shows the current OS version and the URLs used for updates.
 
+``mirror``
+
+    Configure a mirror URL for swupd to use instead of the defaults on the
+    system or compiled into the swupd binary.
+
+    - `-s, --set URL`
+
+        Set the content and version URLs to URL by adding configuration files to
+        <path>/etc/swupd/mirror_contenturl and
+        <path>/etc/swupd/mirror_versionurl
+
+    - `-u, --unset`
+
+        Remove the content and version URL configuration by removing
+        <path>/etc/swupd
+
+    - `-p, --path`
+
+        Set the path prefix under which swupd will operate.
```

Comment:
I would like a little more clarity on what is optional and what is required. So

      -p PATH
       set the optional top level directory for the swupd managed system. If not specified willuse the one in /